### DISCUSSION TITLE
ALT + Tab and WinKey + Tab not working

### DISCUSSION THREAD

**Discussion by Toudahl** [valinet/ExplorerPatcher#1900] [General]
Trying to switch between open windows using ALT + Tab or WinKey + Tab does not work.
> "Task View" also doesn't open even when manually clicked rather than the keyboard shortcut.
 - flipperbw, https://github.com/valinet/ExplorerPatcher/discussions/1900#discussioncomment-6802091

If i restart the explorer.exe it will work for a brief moment.
If i uninstall explorer patcher, it works again
Installing explorer patcher breaks it again.

OS Build: 22621.2134
ExplorerPatcher version: 22621.1992.56.2

Do you need any other information from me to look into this problem?
Im not sure how i can help.

**Comment by Toudahl** [comment]
I had already created an [issue](https://github.com/valinet/ExplorerPatcher/issues/1899), which i closed after seeing https://github.com/valinet/ExplorerPatcher/issues/56

**Comment by flipperbw** [comment]
same thing here. i would say it started yesterday and is borderline unusable now :(
"Task View" also doesn't open even when manually clicked rather than the keyboard shortcut.

**Reply by Toudahl** [reply]
Are you using the same versions as I am? I think this is important information for the developers.
I have also added your experience to my post

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1687

Please close this one.

Work around: set to use windows 11 taskbar instead of windows 10 taskbar.

**Reply by Toudahl** [reply]
While this solves the problem, it also means that my taskbar icons are combined. Which is the only reason i use the patcher :(

I dont see a way to close a discussion. Why do you want it closed?

**Reply by pyrates999** [reply]
It was already reported in the linked issue.  It doesn't help to fix bugs with duplicated issues being open.  Please close this and report on the linked issue.

**Reply by Toudahl** [reply]
This is a discussion not an issue.
If you are a member of this team and you think it should be closed, go ahead.

**Reply by neroux** [reply]
Sorry, but rather the issue should be closed than this thread as https://github.com/valinet/ExplorerPatcher/issues/56 clearly states that issues should not be filed.

**Comment by fwz-git** [comment]
This has been reported for a while now and I had to stop using Explorer Patcher a few weeks ago, because Win+Tab is quite important for my daily work. Was really looking forward to this being fixed in the release, but it still occurs after installing the release from earlier today.

None of the workarounds in other issues worked for me, except the Win11 Taskbar one, which kinda takes away a key feature ...

So I guess we'll have to wait for an upcoming release ...

**Comment by clonsberry** [comment]
Reporting results of testing on my Win11 machine:

- Uninstall and deletion of registry keys (see e.g. [1821](https://github.com/valinet/ExplorerPatcher/issues/1821)) appears to work, at first. Version 22621.1992.56.3. 
	- Problem 1: Start button is not working. Rebooting did not work (`shutdown -r -t 0`). Toggled Start menu to Windows10, then back to Windows11 and things seem to be working now. 
	- Final problem: Use `Logo+Tab` once and then everything is broken again. Uninstalling for now. 
	
:sob: Love this app and can't wait for a fix!

EDIT: Seems like if you only use one desktop you could maybe avoid this issue by never using `Logo+Tab`. Might be worth testing.

**Comment by Cadill** [comment]
WIN+TAB and ALT+TAB still broken for

EP: 22621.1992.56.3
Win 11: 22621.2134

It must be hard to keep up with changes from MS.

Thanks for working on this, much appreciated!

**Comment by pyrates999** [comment]
@Poopooracoocoo this should be closed as it is a duplicate of https://github.com/valinet/ExplorerPatcher/issues/1687

**Reply by MamiyaOtaru** [reply]
not sure discussions _can_ be closed champ

**Reply by neroux** [reply]
Sorry, but rather the issue should be closed than this thread as https://github.com/valinet/ExplorerPatcher/issues/56 clearly states that issues should not be filed.

**Comment by Warlock-Merlin** [comment]
I am also having this issue on Windows 11, I use Alt + Tab all the time I need this fixed I like my freaking task bar at the top. cause when drawing the task bar is always opening up stupid applications every time I am wanting to draw and my wrist is always touching the task bar and opening large programs all the time that I have pinned. My Volume Meter and Display Brightness meter is also gone too.

EP: 22621.1992.56.3
Win 11: 22621.2215

Windows 11 Pro - HP Spectre 16 X360 i7-1260P - 32GB - Intel ARC A370M

**Comment by baldcoder60** [comment]
I'm having the same problem.

**Reply by baldcoder60** [reply]
I will note that ALT-TAB works until I try to use WIN-TAB. Then ALT-TAB quits working until I reboot. WIN-TAB doesn't work at all.

**Reply by Warlock-Merlin** [reply]
when you change from win11 style to win10 style is when everything breaks. no display brightness or volume pop-up or Alt + Tab functionality. Love this tool but this has gotta work. Unless Microsoft has bricked this tool over night.

**Reply by Poopooracoocoo** [reply]
> Unless Microsoft has bricked this tool over night.

Well that's what basically happened. Everything used to work great until Microsoft's cumulative update, KB5029263. In an ideal world Microsoft would never have messed with their functioning taskbar and start menu from Windows 10 to begin with.

**Reply by Warlock-Merlin** [reply]
Whats even more annoying is that blocking updates with Winaero Tweaker in Win11 doesn't even work at all. It only currently bricks Microsoft store apps from being downloaded in W11. Where as in Win 10 Update blocks and MS Store stopped as well which is what I wanted. I use my computer for productivity I don't have time to be dealing with Microsoft pushing out buggy updates that reduce the life on your SSD because of there faulty updates they push out to the mass population and think its okay to oh I don't know brick your windows environment or completely reinstall windows because oh where sorry you install your programs on your separate drive making you have to reformat everything. like Microsoft has completely lost there marbles when it comes to Win11 how about I just go back to Win10 build 1809 everything just worked fine. like I want my taskbar on the top of my screen Microsoft! Restricting the use of the taskbar to only the bottom of the screen is the worst idea when trying to lay your wrist in hand while drawing on your screen. Who draws on their screen without their side of their palm resting on their screen I don't want to launch or populate other applications and touch the screen and make everything wig out because I'm trying to actually do something productive called drawing or rotoscoping in after effects. I am tired of microsoft's terrible operating system, Whatever happened to that terminology called if it's not broke don't fix it? What's the purpose of Windows 11 if the touch screen capability is asinine to use. Why is it that every single time I fold my laptop into tablet mode when I want to type in a field box the keyboard never displays itself automatically like it should. You wouldn't actually tell me you would actually use a touch keyboard that takes up 50% of the screen. and don't get me started on not being able to remove Microsoft Edge I already use Google Chrome I dont need a second version of Chrome Browser installed on my computer using up 2 GB's of storage for an application I don't use.
